catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/security/app/UserApplication.java

```diff
@@ -278,6 +278,7 @@ private Map getSubjectPreferences(Subject subject) {
         .put("serverGenerated", persistentItem.getOrDefault("serverGenerated", "true"))
         .put("when", persistentItem.getOrDefault("when", DateTime.now().toInstant().getMillis()))
         .put("unseen", Boolean.parseBoolean((String) persistentItem.getOrDefault("unseen", "true")))
+        .put("count", persistentItem.getOrDefault("count", -1))
```

Comment:
does anything bad happen when the count is -1?

Reply:
No, we just wanted a non-null value that would be ignored, since the DDF-style notifications are also stored here and won't use count